src/components/multi-action-button/multi-action-button.component.tsx

```diff
@@ -104,38 +104,59 @@ export const MultiActionButton = ({
       );
       let nextIndex = -1;
 
```

Comment:
It looks like you can remove all `ev.preventDefault()` from `handleKeyDown` and add just one at the top of this method because it seems that we always want to prevent default browser behavior when using keyboard on these buttons.
Also this will fix the issue with the screen scrolling when using the Up and Down arrows.